src/sentry_plugins/github/endpoints/webhook.py

```diff
@@ -71,7 +71,7 @@ def __call__(self, organization, event):
 
             try:
                 with transaction.atomic():
-                    c = Commit.objects.create(
+                    c, created = Commit.objects.get_or_create(
                         repository_id=repo.id,
                         organization_id=organization.id,
                         key=commit['id'],
```

Comment:
do you not want `message`, `author`, `date_added` in `defaults`? also, is there a reason you're not using `create_or_update`?